config/initializers/active_job_logger_patch.rb

```diff
@@ -5,6 +5,13 @@
   module ActiveJob
     module Logging
       class LogSubscriber
+        def enqueue(event)
+          info do
+            job = event.payload[:job]
+            "Enqueued #{job.class.name} (Job ID: #{job.job_id}) to #{queue_name(event)} at: #{Time.zone.now}"
```

Comment:
Sure, I can prefix it. I don't see any instances of the other events in the log. I think the "Enqueued" one is the only one we use.